### PR TITLE
UI fixes and PROBLEMS view

### PR DESCRIPTION
**Pull request by NoaBablikiCertora**

### CODE REVIEW

**Comment by NoaBablikiCertora** [comment]
* remove hover from the information icon

* remove default values from duration / loop_iter

* add an option to choose branch in production: customize branch, staging

File: src/results/App.svelte [hunk 1]
```diff
@@ -620,14 +658,18 @@
    * run all the runs that are allowed to run
```
Comment (Biced) [inline src/results/App.svelte:658]: please rewrite using filter

File: src/results/App.svelte [hunk 2]
```diff
@@ -776,58 +818,63 @@
...
+                rs => rs.fileName === runs[index].name,
+              ) !== undefined && pendingQueueCounter > 0}
+              expandedState={$verificationResults.find(
+                vr => vr.name === runs[index].name,
+              ) !== undefined}
+              pendingStopFunc={() => {
+                pendingStopFunc(runs[index])
+              }}
+              runningStopFunc={() => {
+                // $verificationResults = $verificationResults.filter(vr => {
```
Comment (Biced) [inline src/results/App.svelte:856]: please delete comments

File: src/results/App.svelte [hunk 3]
```diff
@@ -776,58 +818,63 @@
...
-          {/key}
+              inactiveSelected={$focusedRun}
+              {setStatus}
+              vrLink={runs[index].vrLink}
+              hide={$hide[index]}
+              pos={$pos}
+              bind:runName={runs[index].name}
+            />
+          </li>
+          <!-- {/key} -->
```
Comment (Biced) [inline src/results/App.svelte:877]: delete comment

**Review by Biced — CHANGES_REQUESTED** [review]
almost corgi keshet

**Pull request merged**